.github/workflows/generate_baseline_profile.yml

```diff
@@ -0,0 +1,95 @@
+name: Generate Baseline Profile
+
+on:
+  workflow_dispatch:
+  workflow_call:
+    secrets:
+      token:
+        required: true
+
+jobs:
+  test:
+    runs-on: macos-latest
+    strategy:
+      matrix:
+        jdk-version: [ "11" ]
+        ndk-version: [ "21.4.7075529" ]
+        api-level: [ 29 ]
```

Comment:
Should we target a more current API version? Seems like the recommended right now is 33